deploy/helm/trickster/templates/service.yaml

```diff
@@ -1,12 +1,17 @@
 apiVersion: v1
 kind: Service
 metadata:
-{{- if .Values.service.annotations }}
+{{- if or .Values.service.annotations .Values.prometheusScrape }}
```

Comment:
Did you also want to include this set of changes in this PR as well since it was in #395 already?

Reply:
should be able to manage this merge as is; noting that the only effective change in this pr is the renaming of `metrics` to `http-metrics`